extras/nginx-mtls-auth/model_server.conf.template

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+map $http_upgrade $connection_upgrade {
+    default upgrade;
+    ''      close;
+}
+
+server {
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Scheme $scheme;
+    proxy_set_header Host $http_host;
+    listen              NGINX_LISTEN_GRPC_PORT ssl http2;
+    ssl_certificate     /certs/server.pem;
+    ssl_certificate_key /certs/server.key;
+    ssl_client_certificate /certs/client_cert_ca.pem;
+    ssl_verify_client on;
+    ssl_verify_depth 1;
+    client_max_body_size 100M;
```

Comment:
1GB might be safer and in line with ovms limits

Reply:
changed